apps/perf-test-react-components/just.config.ts

```diff
@@ -4,4 +4,4 @@ import { preset, task, series } from '@fluentui/scripts-tasks';
 preset();
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26893&commitId=7395ee94911f5a38c237737ea28dc1dc96f52fb2&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu.Menu Example Divider Horizontal.chromium.png|76| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>